JS-Algorithms/challenges.js

```diff
@@ -118,14 +164,29 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let a = 1;
```

Comment:
please use more descriptive variable names, imagine that you are writing code for other devs and not only for yourself, so the easier to read/understand the better

Reply:
Fixed